modules/account-lib/src/coin/dot/iface.ts

```diff
@@ -12,6 +12,7 @@ export enum MethodNames {
   TransferKeepAlive = 'transferKeepAlive',
   Unbond = 'unbond',
   Anonymous = 'anonymous',
+  Chill = 'chill',
```

Comment:
what do you think about using `unnominate` as the MethodName so that it's easy to understand it's used to select the `UnnominateBuilder`? If you think chill is appropriate then thats fine with me as well.

Reply:
I wasn't too sure about the naming conventions so I followed how stake/unstake did it. For stakingBuilder the method was just bond, and for unstakeBuilder the method is unbond. In this case I used chill to follow the previous pattern.